### DISCUSSION TITLE
Welcome to immich Discussions!

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ippocratis** [comment]
Been desperately searching for a selfhosted photo manager with a worthy android app
There are not much
Well there are 3 that I'm aware of
Librephotos
photoprism
Piwigo

All great services but with buggy slow unfinished android apps.

Immich comes with the android app first in mind
Testing it and it was rediculusly fast,  like loading local photos fast it was mindblowing.

I'm looking forword for some basic features like importing my remote photos to get implemented so I can migrate to immich

I'm excited

**Reply by alextran1502** [reply]
Thank you for your kind words, they give me a lot of motivation for this project!
